patches/lua-tree.patch

```diff
@@ -414,11 +409,7 @@ index 6c92098cc..c3c3c12fe 100644
  local type          = type
  local pairs         = pairs
  local ipairs        = ipairs
-@@ -33,34 +26,11 @@ local fmt           = string.format
- local find          = string.find
- local gsub          = string.gsub
- local split         = pl_stringx.split
--local re_find       = ngx.re.find
```

Comment:
My advice is keep the patch as it was (e.g. remove this commit); this PR was in great shape and only needed things documented.

Reply:
I stand corrected on this one; resolving thread.